app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
     <string name="app_name">NewsApp</string>
     <string name="noNews">No News Was Found</string>
-    <string name="baseuri">https://content.guardianapis.com/</string>
+    <string name="baseuri">http://content.guardianapis.com/search</string>
```

Comment:
iirc: I had initially forgot the /search in the URL and so i just added that - it was still not working, so I doublt checked everything and the only difference in what i had written vs what the API Was telling me the baseURI was was the "s" so i dropped that.

That also didnt work because i was using "api_key" instead of "api-key".  I didnt even think to try and re-add the s.